package.json

```diff
@@ -23,18 +23,18 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "browserify-mime": "~1.2.9",
+    "browserify-mime": "^1.2.9",
     "extend": "^3.0.2",
-    "json-edm-parser": "0.1.2",
-    "md5.js": "1.3.4",
-    "readable-stream": "~2.0.0",
+    "json-edm-parser": "~0.1.2",
```

Comment:
Why some dependencies use `~` while some others use `^`?

Reply:
Per offline sync, packages starting `0` major version will keep using `~` to avoid breaking changes in minor version update.